spec/models/miq_report/import_export_spec.rb

```diff
@@ -18,7 +18,7 @@
     before do
       @user_admin = FactoryBot.create(:user_admin)
       report_string = MiqReport.export_to_yaml([@old_report.id], MiqReport)
-      @new_report   = YAML.load(report_string).first
+      @new_report   = YAML.load(report_string).first["MiqReport"]
```

Comment:
These report changes should be moved to a separate PR to keep this GenericObject focused.